addons/html_editor/static/src/core/format_plugin.js

```diff
@@ -134,8 +134,12 @@ export class FormatPlugin extends Plugin {
             if (!formatsSpecs[format].removeStyle || !this.hasSelectionFormat(format)) {
                 continue;
             }
-            this.formatSelection(format, { applyStyle: false });
+            this._formatSelection(format, { applyStyle: false });
         }
+        for (const cb of this.resources["removeFormat"]) {
```

Comment:
`for (const cb of this.resources["removeFormat"] || []) { ...}`
In case no plugin adds something to this resource (ex: an editor without the ColorPlugin), you would be trying to iterate over `null`, which would crash.

Reply:
And I personally prefer `callback` over `cb` (no abbreviation). Both of them are currently used in html_editor, so it's fine if you prefer to keep it `cb`.